src/routes/AppRouter.js

```diff
@@ -2,11 +2,13 @@ import React from 'react';
 import { Route, Switch } from 'react-router-dom';
 import LandingPage from '../pages/loginPage';
 import Homepage from '../pages/hompage';
+import Card from '../components/registration/index';
 
 const Routes = () => (
   <Switch>
     <Route path="/" exact component={Homepage} />
     <Route path="/login" exact component={LandingPage} />
+    <Route path="/card" exact component={Card} />
```

Comment:
I am still unsure if this is the right way this should work, how does this component know it should be rendered when the screensize is small? and not render when the screen size is large